### DISCUSSION TITLE
print qr-code use vue-html-to-paper

### DISCUSSION THREAD

**Discussion by amin7mazaheri** [npm/rfcs#252] [Question]
I am using Vue-html-to-paper for printing QR but it returns an empty page, is there any way to solve it ???

**Comment by ljharb** [comment]
You'll need to file that issue on their repo: https://github.com/mycure-inc/vue-html-to-paper this repo is for RFCs for npm itself.

**Comment by MareSeestern** [comment]
@amin7mazaheri 

Solved it by adding the prop "render-as" and changing its value to "svg".

` <qrcode-vue render-as="svg" :value="example.com" size="200" level="H" />`

**Reply by Deusleon** [reply]
Thank you, solved it.
